server/routes/api/variable_group.py

```diff
@@ -36,7 +36,8 @@ def get_variable_group_info():
   """
   dcid = request.args.get("dcid")
   entities = request.args.getlist("entities")
-  result = dc.get_variable_group_info(dcid, entities)
+  resp = dc.get_variable_group_info([dcid], entities)
+  result = resp.get("data")[0].get("info")
```

Comment:
add checks or default return values for the dereferences?

Reply:
Done